ui/push-health/CommitHistory.jsx

```diff
@@ -34,66 +34,65 @@ class CommitHistory extends React.PureComponent {
         revisionCount,
       },
       revision,
+      currentRepo,
     } = this.props;
     const { clipboardVisible } = this.state;
-    const repoModel = new RepositoryModel(repository);
+    const parentRepoModel = new RepositoryModel(parentRepository);
+    const parentLinkUrl = exactMatch
+      ? `${getJobsUrl({
+          revision: parentPushRevision,
+          repo: parentRepository.name,
+        })}`
+      : parentRepoModel.getRevisionHref(parentSha);
 
     return (
       <React.Fragment>
         <h5>Parent Push</h5>
-        {!exactMatch && (
-          <div className="ml-4">
-            <div
-              className="mb-2 ml-3"
-              onMouseEnter={() => this.showClipboard(true)}
-              onMouseLeave={() => this.showClipboard(false)}
-            >
-              <Clipboard
-                description="full hash"
-                text={parentSha}
-                visible={clipboardVisible}
-              />
-              <a
-                href={repoModel.getRevisionHref(parentSha)}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {parentSha}
-              </a>
+        <div className="ml-4">
+          {!exactMatch && (
```

Comment:
Apologies that this is a mish-mash.  What I was doing before was showing one set of elements if it wasn't an exact match (I couldn't find a push in Treeherder for the parent commit) and when I could find a TH Push.  What I'm doing here is merging them so it's one code-path, and only showing the `Alert` when they don't match.  

Then later, on line 79, if we found a Push, I show the `parentPushRevision` which links to Treeherder.  If not, I just show the `parentSha` which links to HG.